docs/contributing.md

```diff
@@ -1,122 +1,39 @@
 
-Next, clone your forked repository to your local machine and set up the main repository as an "upstream" repository:
+- **Use maintainer privileges wisely**
 
-```sh
-git clone git@github.com:$user_name/PrairieLearn.git
-cd PrairieLearn
-git remote add upstream git@github.com:PrairieLearn/PrairieLearn.git
-```
+  - Do not approve or merge changes you do not fully understand.
+  - If you’d like to understand a PR better, ask questions rather than making assumptions.
 
-This means that you now have three key repositories to keep track of:
+- **Merging responsibilities**
 
-- **Upstream:** the main PrairieLearn repository on GitHub. You can't directly push to this, but this is where you will pull other people's commits from.
-- **Origin:** your forked repository on GitHub. You can push and pull directly to this.
-- **Local:** your clone of origin on your local computer. This is where you commit before pushing to origin.
+  - If a maintainer opens a PR, that person is responsible for merging it after it has been approved.
+  - For PRs from non-maintainers, the primary reviewer is responsible for merging the PR. If there are outstanding nitpicks or unresolved discussions, consult with the author before merging.
 
-**Important note:** Don't ever commit anything to the `master` branch. You should do all your work in branches and issue pull requests from these branches back to the main PrairieLearn repository.
+- **Taking over abandoned PRs**
+  - If a PR appears abandoned, another maintainer may choose to take it over.
+  - The new maintainer assumes responsibility for responding to feedback and ultimately merging.
+  - Use discretion when deciding to take over a PR; consider reaching out to the original author first.
 
-**Updating your fork from upstream:** Getting new changes from upstream is a two-step process. First you pull from upstream to your local repository, then you push the new changes back to your origin. (There is no direct communication between upstream and your forked origin repository.) Note that for this to work it is critical that you have never committed anything to your master branch. The procedure to pull and then push is:
+### Code review guidelines
 
-```sh
-git checkout master      # make sure you are on the master branch
-git pull upstream master # pull new changes from upstream
-git push origin master   # push the new changes back up to your fork (origin)
-```
+- **Review comments**
 
-See also the GitHub Help pages on [Syncing a fork](https://help.github.com/articles/syncing-a-fork/) and [Pushing to a remote](https://help.github.com/articles/pushing-to-a-remote/).
+  - It is acceptable to approve a PR and still leave comments. The expectation is that these comments will be addressed by the PR author before the PR is merged.
+  - Small, subjective suggestions ("nits") are acceptable in reviews. Clearly label nits (e.g., "Nit: consider renaming this for clarity.") to distinguish them from required changes.
```

Comment:
I think it's worth encouraging more comments, as long as everything is constructive the more comments there are the better the end product usually is, unless they are excessively pedantic.

IMO it's also worth encouraging maintainers to review PRs even without request. Lack of overall review resources have been a pain point on this project, and encouraging people to review additional PRs when they have time help alleviate this.

Reply:
How do the changes from ce81502 and de7f524 look?

Reply:
Changes look good 👍🏽